### PR TITLE
move baidu-ui，replace by antd

### PR DESCRIPTION
**Pull request by wanganjuan**
move baidu-ui.
and replace bu antd
---
first commit can compile, but some ui need to modify to adapt.
If more people are needed，
I'll continue to fix it。
---
It is currently in the development stage, and there are many files modified. In the final form, all the changes will be aggregated into a single file, uploaded to npmjs.com. In this way, for different branches, you only need to download this package and import it.

### CODE REVIEW

**Comment by imbajin** [comment]
Thanks a lot for the contribution, and we'll push current project into `Apache Foundation`, your efforts will be needed by the **whole community**. 

We will try our best to see the relevant review matters and follow-up the PR, feel free to update it

**Comment by imbajin** [comment]
And we need uncomment these line to build the whole project together (in [dist/pom.xml](https://github.com/hugegraph/hugegraph-hubble/blob/7b53a4cd3e3b23f0e95b53bb40c2f4ce75b25555/hubble-dist/pom.xml#L67))

image

With no building problems in FE, we could use `mvn clearn package` in the root directory to build the project

**Comment by wanganjuan** [comment]
> Thanks a lot for the contribution, and we're pushing current project into `Apache Foundation`, your efforts will be needed by the **whole community**.
> 
> We will try our best to see the relevant review matters and follow-up the PR, feel free to update it
> 
> And a personal question, why we choose `ant-ui` (opens source, popular, unified API or? ) I don't know much about the front-end UI framework and hope that the framework chosen will be **easy for most people** to handle so that other community contributors can follow it

baidu-ui, ant.design ui (antd) are all based on react. antd is currently the most popular react ui library. You can convert baidu ui components into antd ui components. This is a feasible solution.
image

**Comment by wanganjuan** [comment]
> And we need uncomment these line to build the whole project together
There is currently no focus on overall development and deployment。I just modify the hubble-fe project, the independent front end. You can configure the address of the back-end interface in the config-overrides.js of this project.
image

File: hubble-fe/README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# HugeGraph-Hubble
-
-Front-end implementation of HugeGraph
\ No newline at end of file
+1: add UI/index.tsx file and replace @baidu-ui to @ui
+2: GraphManagementList.tsx QueryAndAlgorithmLibrary.tsx 的 dropdown 重写
+3：Embedded 组件的 的修改
```
Comment (javeme) [inline hubble-fe/README.md:3]: we can create a new file CHANGELOG.md and move this content into it

File: hubble-fe/config-overrides.js [hunk 2]
```diff
@@ -1,7 +1,31 @@
...
+      javascriptEnabled: true,
+    }),
+    addWebpackAlias({
+      '@ui': require('path').resolve(__dirname, './src/components/common/UI'),
+    })
+  ),
+  devServer: overrideDevServer(
+    addProxy(),
+    watchAll()
+)
```
Comment (javeme) [inline hubble-fe/config-overrides.js:30]: align with devServer?

File: hubble-fe/src/components/graph-management/GraphManagementList.tsx [hunk 3]
```diff
@@ -422,18 +397,26 @@ const GraphManagementListItem: React.FC<
...
+          ) : ( <Menu onClick={handleDropdownClick(index)}>
+          <Menu.Item key="edit" disabled>编辑</Menu.Item>
+          <Menu.Item key="delete">删除</Menu.Item>
+        </Menu>)}
           title="更多"
           size="medium"
           trigger={['click']}
           width={78}
-          onHandleMenuClick={handleDropdownClick(index)}
+          // onClick={handleDropdownClick(index)}
```
Comment (javeme) [inline hubble-fe/src/components/graph-management/GraphManagementList.tsx:413]: unused?

File: hubble-fe/src/components/graph-management/GraphManagementList.tsx [hunk 4]
```diff
@@ -422,18 +397,26 @@ const GraphManagementListItem: React.FC<
...
           disabled={
             graphManagementStore.showCreateNewGraph === true ||
             (graphManagementStore.selectedEditIndex !== null &&
               graphManagementStore.selectedEditIndex !== index)
           }
-        />
+        >更多</Dropdown.Button>
```
Comment (javeme) [inline hubble-fe/src/components/graph-management/GraphManagementList.tsx:419]: prefer to split into 3 lines

File: hubble-fe/src/components/graph-management/data-analyze/QueryAndAlgorithmLibrary.tsx [hunk 5]
```diff
@@ -321,21 +322,17 @@ export const GremlinQuery: React.FC = observer(() => {
...
-              }}
+              }}>
+              <Menu.Item key="query">执行查询</Menu.Item>
+              <Menu.Item key="task">执行任务</Menu.Item>
+            </Menu>)}
+              trigger={['click']}
+
+              // onHandleMenuClick={(e: any) => {
+              //   dataAnalyzeStore.setQueryMode(e.key);
+              // }}
```
Comment (javeme) [inline hubble-fe/src/components/graph-management/data-analyze/QueryAndAlgorithmLibrary.tsx:335]: ditto

File: hubble-fe/src/fix.less [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+  border-left: 0!important
+}
+
+.new-fc-one-input-group {
+  display: flex
+}
+
+.new-fc-one-input-group .new-fc-one-input {
+  border-radius: 0
+}
```
Comment (javeme) [inline hubble-fe/src/fix.less:83]: expect a blank line at the end of file

File: hubble-fe/src/fix.less [hunk 7]
```diff
@@ -0,0 +1,83 @@
+.new-fc-one-input-count-visible {
```
Comment (javeme) [inline hubble-fe/src/fix.less:1]: can we choose a better file name for fix.less?

File: hubble-fe/src/index.less [hunk 8]
```diff
@@ -70,8 +67,7 @@ body {
       url(./assets/imgs/ic_biaoge_hover.svg)
       url(./assets/imgs/ic_biaoge_pressed.svg)
       url(./assets/imgs/ic_json_normal.svg) url(./assets/imgs/ic_json_hover.svg)
-      url(./assets/imgs/ic_json_pressed.svg)
-      url(./assets/imgs/ic_close_16.svg)
+      url(./assets/imgs/ic_json_pressed.svg) url(./assets/imgs/ic_close_16.svg)
```
Comment (javeme) [inline hubble-fe/src/index.less:70]: align with the previous line

**Review by javeme — COMMENTED** [review]
Thanks for your contribution, I think it's a good idea to port hubble to antd, which is already an open source project.
There are some small comments, and we will do some testing later.

File: hubble-fe/src/components/graph-management/GraphManagementList.tsx (hunk 3 above)
Comment (wanganjuan) [reply hubble-fe/src/components/graph-management/GraphManagementList.tsx:413]: The exposed parameters of the two components are different, and some need to be rewritten. Part of the commented code is not needed, I will format and delete it later.

File: hubble-fe/README.md (hunk 1 above)
Comment (wanganjuan) [reply hubble-fe/README.md:3]: ok

File: hubble-fe/CHANGELOG.zh-CN.md [hunk 9]
```diff
@@ -1,133 +1,7 @@
-## 1.6.0 (2020-08-14)
+#### ui modify
```
Comment (javeme) [inline hubble-fe/CHANGELOG.zh-CN.md:1]: can we keep the history change log in this file?

File: hubble-fe/.gitignore [hunk 10]
```diff
@@ -0,0 +1,23 @@
+.DS_Store
+node_modules
+/dist
+
+
```
Comment (javeme) [inline hubble-fe/.gitignore:5]: can remove a blank line

File: hubble-fe/src/index.less (hunk 8 above)
Comment (javeme) [inline hubble-fe/src/index.less:70]: need to wrap line before the second "url"

Comment (wanganjuan) [reply hubble-fe/src/index.less:70]: ok

File: hubble-fe/config-overrides.js [hunk 11]
```diff
@@ -1,7 +1,34 @@
...
 
-module.exports = override(
-  addLessLoader({
-    javascriptEnabled: true
-  })
-);
+const addProxy = () => (configFunction) => {
+  configFunction.proxy = {
+    '/about': {
+      target: 'http://172.31.164.8:9898',
```
Comment (coderzc) [inline hubble-fe/config-overrides.js:12]: change to http://127:0.0.1:8088 ?

File: hubble-fe/config-overrides.js [hunk 12]
```diff
@@ -1,7 +1,34 @@
...
+      changeOrigin: true
+    },
+    '/api': {
+      target: 'http://172.31.164.8:9898',
```
Comment (coderzc) [inline hubble-fe/config-overrides.js:16]: ditto

**Review by coderzc — CHANGES_REQUESTED** [review]

**Review by coderzc — APPROVED** [review]

**Comment by wanganjuan** [comment]
new pr: https://github.com/hugegraph/hugegraph-hubble/pull/344

**Pull request closed without merging**